google/cloud/aiplatform/utils/worker_spec_utils.py

```diff
@@ -86,6 +91,10 @@ def spec_dict(self) -> Dict[str, Union[int, str, Dict[str, Union[int, str]]]]:
             spec["machine_spec"]["accelerator_type"] = accelerator_type
             spec["machine_spec"]["accelerator_count"] = self.accelerator_count
 
+        if self.container_uri == REDUCTION_SERVER_CONTAINER_URI:
```

Comment:
For now, I think it may be more extendable to add `container_spec` whenever `self.container_uri` is populated.